lfshttp/client.go

```diff
@@ -209,6 +209,12 @@ func (c *Client) sshResolveWithRetries(e Endpoint, method string) (*sshAuthRespo
 	var sshRes sshAuthResponse
 	var err error
 
+	uc := config.NewURLConfig(c.gitEnv)
+	if val, ok := uc.Get("lfs", e.OriginalUrl, "sshprotocol"); ok && val != "auto" && val != "git-lfs-authenticate" {
+		tracerx.Printf("skipping SSH-HTTPS hybrid protocol connection by request")
```

Comment:
+1 for adding this. While adding pure-SSH support to GitLab, I've had to take care to make sure git-lfs isn't falling back to `git-lfs-authenticate` in the event that `git-lfs-transfer` fails, resulting in me not noticing that my GitLab `git-lfs-transfer` code doesn't actually work. It would be nice to have a way to ensure that only `git-lfs-transfer` is ever attempted.